### PR TITLE
Chain multiple redemption strategies

### PR DESCRIPTION
**Pull request by vminkov**
## Description

Adding the configuration ability to chain multiple redemption strategies for a liquidation. 

## Related pull request(s)

https://github.com/Midas-Protocol/contracts/pull/326

### CODE REVIEW

File: packages/sdk/src/chainConfig/redemptionStrategies.ts [hunk 1]
```diff
@@ -1,71 +1,118 @@
...
   [SupportedChains.evmos]: {
-    "0x -- Kinesis": RedemptionStrategy.CurveLpTokenLiquidatorNoRegistry,
+    "0x -- Kinesis": [RedemptionStrategyContract.CurveLpTokenLiquidatorNoRegistry, "ignored"],
   },
   [SupportedChains.ganache]: {},
   [SupportedChains.neon_devnet]: {},
   [SupportedChains.polygon]: {},
 };
 
+function underlying(assets: SupportedAsset[], symbol: string): string {
```
Comment (carlomazzaferro) [inline packages/sdk/src/chainConfig/redemptionStrategies.ts:112]: nice. maybe we place it somewhere else, possibly in the `chainDeploy/helpers/utils.ts` ?

**Review by carlomazzaferro — APPROVED** [review]

File: packages/sdk/src/chainConfig/addresses.ts [hunk 2]
```diff
@@ -62,6 +72,8 @@ const chainSpecificAddresses: ChainSpecificAddresses = {
     UNISWAP_V2_ROUTER: "0x10ED43C718714eb63d5aA57B78B54704E256024E",
     UNISWAP_V2_FACTORY: "0x985BcA32293A7A496300a48081947321177a86FD",
     PAIR_INIT_HASH: ethers.utils.hexlify("0x00fb7f630766e6a796048ea87d01acd3068e8ff67d078148a3fa3f4a84f69bd5"),
+    STABLE_TOKEN: moonbeamAssets.find((a) => a.symbol === assetSymbols.madUSDC)!.underlying,
```
Comment (carlomazzaferro) [inline packages/sdk/src/chainConfig/addresses.ts:75]: these are the Nomad assets -- we likely want to use the multichain's USDC

File: packages/sdk/src/chainConfig/addresses.ts [hunk 3]
```diff
@@ -94,6 +112,8 @@ const chainSpecificAddresses: ChainSpecificAddresses = {
     UNISWAP_V2_ROUTER: "0xa5E0829CaCEd8fFDD4De3c43696c57F7D7A678ff",
     UNISWAP_V2_FACTORY: "0x5757371414417b8C6CAad45bAeF941aBc7d3Ab32",
     PAIR_INIT_HASH: ethers.utils.hexlify("0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f"),
+    STABLE_TOKEN: polygonAssets.find((a) => a.symbol === assetSymbols.BUSD)!.underlying,
```
Comment (carlomazzaferro) [inline packages/sdk/src/chainConfig/addresses.ts:115]: Polygon's most widely used stable is probably USDC, not BUSD

File: packages/sdk/src/chainConfig/addresses.ts (hunk 2 above)
Comment (vminkov) [reply packages/sdk/src/chainConfig/addresses.ts:75]: I saw have still have a uniswap pair with WGLMR and should have worked for the liqui bots

**Review by carlomazzaferro — APPROVED** [review]

File: packages/sdk/src/chainConfig/addresses.ts (hunk 2 above)
Comment (carlomazzaferro) [reply packages/sdk/src/chainConfig/addresses.ts:75]: well, it would but there's nothing being borrowed there

**Pull request merged**